state.json

```diff
@@ -71,6 +71,15 @@
         "commands": [
           "https://status.mozilla.org/"
         ]
+      },
+      {
+        "name": "tor",
+        "commands": [
+          "http://yyz-conf-free.paas.allizom.org/",
+          "http://yyz-conf-free.paas.allizom.org/list",
+          "http://departures.moo.mx/index.html#mozilla-toronto",
+          "http://forecast.io/#/f/43.6474,-79.3942"
```

Comment:
This gives near term forecast and weather alerts formatted to read well from 10 feet. You don't have to take it, but I think it's nicer than the default website.

![screencap](https://dl.dropboxusercontent.com/u/105227/toronto.png)